README.md

```diff
@@ -64,7 +64,7 @@ Here are all the samples which are deployed on Microsoft Azure and you can try i
 |9| Universal bots  | Teams catering bot demonstrating how to use Universal bots in Teams | [View](samples/bot-teams-catering/csharp) |  | |
 |10| Sequential workflow adaptive cards | Demonstrating on how to implement sequential flow, user specific view and upto date adaptive cards in bot. | [View][sequential#workflow#csharp] | [View][sequential#workflow#js]
 |11| Channel messages with RSC permissions | Demonstrating on how a bot can receive all channel messages with RSC without @mention. | [View][messageswithrsc#csharp] |[View][messageswithrsc#js]
-
+|12| Archive groupchat messages | Demonstrating on how a bot can archive groupchat messages and send it to user as a file. |  |[View][fetchgroupchatmessages#nodejs]|
```

Comment:
Let's move this under Scenario specific samples: https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/v-nikija/bot-groupchat-rsc-nodejs#scenario-specific-samples